homeassistant/components/senz/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "senz",
+  "name": "nVent RAYCHEM SENZ",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/senz",
+  "requirements": ["aiosenz==1.0.0"],
+  "dependencies": ["auth", "http"],
```

Comment:
```suggestion
  "dependencies": ["auth"],
```

Reply:
Aah... Thanks :+1: 
I missed that development and was confused. :smile: